1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -88,6 +88,99 @@ \section{Produktumgebung}
 
 \section{Funktionale Anforderungen}
 
+\subsection{Wahlfunktion fuer einen Waehler bereitstellen}
+\begin{itemize}
+	\item Ziel: Vote fuer einen Kandidat geben.
+	\item Subsystem: Die GUI bildet die Schnittstelle zwischen dem Nutzer und dem Produkt.
+	\item Das System ueberprueft, ob der Wahlprozess eingeleitet ist.	
+	\begin{enumerate}
+		\item Falls die Wahlen eingeleitet sind: Das System bietet mehrere Kandidaten zur Auswahl. Der Wähler wählt einen Kandidaten. Das System fordert Sie auf, die Auswahl zu bestätigen. Wenn der Wähler seine Wahl bestätigt, berücksichtigt das System die Wahl des Benutzers und fügt der Einheit Informationen in Block-Chain hinzu. Das System schließt seine Arbeit ab.
+		\item Falls die Wahlen nicht eingeleitet sind: Das System gibt dem Benutzer eine Informationsnachricht "Wahlen haben nicht begonnen". Das System schließt seine Arbeit ab.
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Netzwerkadministratorfunktionalitaet}
+\begin{itemize}
+	\item Ziel: den Wahlprozess zu verwalten.
+	\item Subsystem: : Verwaltung der Wahlen.
+	\item Das System ueberprueft, ob der Wahlenprozess begonnen ist.	
+	\begin{enumerate}
+		\item Falls die Wahlen eingeleitet sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Beenden die Wahl. (Falls diese Option ausgewählt ist, gehen Sie zu 5.4)
+		\end{enumerate}
+		\item Falls die Wahlen nicht eingeleitet sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Den Kandidat hinzufuegen. Falls diese Option ausgewählt ist, gehen Sie zu 5.6)
+			\item Den Kandidat loeschen Falls diese Option ausgewählt ist, gehen Sie zu 5.7)
+		\end{enumerate}
+		\item Falls die Wahlen schon abgeschlossen sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Neue Wahlen beginnen (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.2.2)
+		\end{enumerate}
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Zaehlen der Votes}	
+\begin{itemize}
+	\item Ziel: berechnen die Anzahl der von Waehler gegebenen Votes.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt Informationen über jeden Kandidaten (Name, Nachname, Foto des Kandidaten) und die Anzahl der von jedem der Wähler erhaltenen Votes aus. Der Netzwerkadministrator nach der Überprüfung der Informationen geht zurück (siehe 5.2.1 oder 5.2.3). 
```

Comment:
Fotos von Kandidaten würde ich definitiv weglassen, @Schaback @tinytiny123 @achimkr sollten Nutzer eindeutig über name, nachname identifiziert werden? wenn ja steht das eventuell in konflikt damit, dass in der Section Produktdaten keine Daten über die Nutzer gespeichert werden? (das ist mein Wissensstand zu den Produktdaten)

Reply:
Der Wähler wird eindeutig durch sein X.509 Zertifikat identifiziert, Optionale Bilder der Kandidaten finde ich nicht so schlimm...

Reply:
Denke Fotos können wir machen. Müsste dann halt vermutlich byteweise in den ledger geschrieben werden und am andern ende wieder dekodiert werden aber das geht denk ich klar.